javascript/package.json

```
@@ -156,25 +163,37 @@
       "rollup@>=4.0.0 <4.59.0": ">=4.59.0",
       "trim-newlines@<4.0.1": ">=4.0.1",
       "flatted@<3.4.2": ">=3.4.2",
-      "liquidjs@<10.26.0": ">=10.26.0",
+      "liquidjs@<10.27.1": "10.27.1",
```

Biome (2.5.5)

[error] The key liquidjs@<10.27.1 was already declared.

(lint/suspicious/noDuplicateObjectKeys)